test/functional/p2p_invalid_messages.py

```diff
@@ -124,13 +124,9 @@ def test_checksum(self):
     def test_size(self):
         conn = self.nodes[0].add_p2p_connection(P2PDataStore())
         with self.nodes[0].assert_debug_log(['']):
-            msg = conn.build_message(msg_unrecognized(str_data="d"))
-            cut_len = (
-                4 +  # magic
-                12  # msgtype
-            )
-            # modify len to MAX_SIZE + 1
-            msg = msg[:cut_len] + struct.pack("<I", 0x02000000 + 1) + msg[cut_len + 4:]
+            # Create a message with PAYLOAD SIZE = MAX_SIZE + 1
```

Comment:
in commit 793aa7a2e74460bdb504e8f9e2c642b6e607015c:

The comment is incorrect. MAX_SIZE is not equal to VALID_DATA_LIMIT

Reply:
Ok, agreed.  No need to introduce an undefined term.